dspace-api/src/main/java/org/dspace/discovery/configuration/DiscoveryConfigurationService.java

```diff
@@ -20,9 +21,18 @@
  */
 public class DiscoveryConfigurationService {
 
+    private Map<String, List<String>> extraConfigurationMapping = new HashMap<String, List<String>>();
```

Comment:
This configuration isn't used, the getters & setters aren't called, can you remove this variable.

Reply:
removed, see https://github.com/4Science/DSpace/pull/65/commits/cc11f4955677eaedfae70c54eb08e940bb6067fe